src/main/java/org/cactoos/collection/CollectionEnvelope.java

```diff
@@ -147,4 +147,48 @@ public String toString() {
         return this.col.value().toString();
     }
 
+    @Override
+    public final boolean equals(final Object other) {
```

Comment:
@Vatavuk this `equals` is missing the null-check

Reply:
@llorllale fixed in d4538f8